### ISSUE TITLE
Button to jump to replies

### ISSUE THREAD

**Issue by AnisTaluqdar** [vector-im/element-android#3822]
![41234183](https://user-images.githubusercontent.com/41234183/129018030-2632d512-22e6-4d74-bc96-128b3efa0360.jpg)
It would be great when someone replies to you or tags you in the group it’s showing @ button and you can find the chat easily.

Thanks

**Comment by fariszr** [comment]
+1

**Comment by HarHarLinks** [comment]
I think it would fit the current design better on the top of the screen next to "Jump to first unread", but is generally a good idea

**Comment by fariszr** [comment]
> I think it would fit the current design better on the top of the screen next to "Jump to first unread", but is generally a good idea

i think it should be similar to telegram.
having it at the top of the screen is difficult to reach for most mobile users

**Comment by AnisTaluqdar** [comment]
> I think it should be similar to telegram.


Yeah, exactly. This @ option for telegram is great for long conversations.

**Comment by HarHarLinks** [comment]
perhaps element's "go to first unread message" should be reworked to be conveniently in the bottom right as well?

also ftr, schildichat has an option "open rooms at first unread message" which element could backport.

**Comment by AnisTaluqdar** [comment]
https://wabetainfo.com/whatsapp-messenger-beta-for-ios-22-1-71-whats-new/#more-11775

WhatsApp working with it

**Comment by AnisTaluqdar** [comment]
Example : Telegram
https://user-images.githubusercontent.com/41234183/150698863-556a90b0-5cbd-434e-adcc-f25097803380.mp4
